var/spack/repos/builtin/packages/neurodamus-base/package.py

```diff
@@ -0,0 +1,43 @@
+##############################################################################
+# Copyright (c) 2013-2018, Lawrence Livermore National Security, LLC.
+# Produced at the Lawrence Livermore National Laboratory.
+#
+# This file is part of Spack.
+# Created by Todd Gamblin, tgamblin@llnl.gov, All rights reserved.
+# LLNL-CODE-647188
+#
+# For details, see https://github.com/spack/spack
+# Please also see the NOTICE and LICENSE files for our notice and the LGPL.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU Lesser General Public License (as
+# published by the Free Software Foundation) version 2.1, February 1999.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the IMPLIED WARRANTY OF
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the terms and
+# conditions of the GNU Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA 02111-1307 USA
+##############################################################################
```

Comment:
New license is way more compact…

Reply:
Indeed. I didn't touch those files (`neurodamus-base` and `neurodamus`) though. I had actually deleted them but brought them back since we may have to live with previous neurodamus in parallel for a bit. 
So I would keep them "as is" until we drop them.

Reply:
@pramodk Or shall we delete right away? and push users to move and we update all tests accordingly?

Reply:
neurodamus-base and neurodamus are dependent...so we have delete them together. But before that we should get green signal from users for new structure...we should send the email to few users tomorrow (to review new structure)